lib/ansible/modules/cloud/vmware/vmware_host.py

```diff
@@ -43,16 +44,27 @@
   esxi_username:
     description:
     - ESXi username.
-    required: yes
```

Comment:
Add note here saying, "esxi_username is no longer required parameter from version 2.5"

This is to maintain backward compatibility.

Reply:
done